mainlyetcetera/fitlit
# Feature/activity repo tests

### What does this PR do?
- adds tests for the `ActivityRepo` class
### Is this a feature, a fix, or a refactor?
- feature
### Where should the reviewer start?
- `test/ActivityRepo-test.js`
### How should this be tested?
- pull down to branch (not main)
- run `npm test test/ActivityRepo-test.js`
- observe all tests populating
